src/app/game-state/main-loop.service.ts

```diff
@@ -20,8 +21,9 @@ export interface MainLoopProperties {
   pause: boolean;
   bankedTicks: number;
   totalTicks: number;
-  useBankedTicks: boolean;
-  scientificNotation: boolean;
+  useBankedTicks: boolean,
+  scientificNotation: boolean
```

Comment:
Heh, how does this not complain when you compile? punctuate this line please.

Reply:
I'm not sure why it wasn't complaining, but I've updated it. I've been testing via "npm start" since the full build takes a really long time.